src/main/resources/application-folio.yml

```diff
@@ -3,8 +3,8 @@ mod-linked-data:
     page-size: 1000
   kafka:
     topic:
-      work-search-index: linked-data.work
-      instance-ingress: inventory.instance_ingress
+      work-search-index: ${KAFKA_WORK_SEARCH_INDEX_TOPIC:linked-data.work}
+      instance-ingress: ${KAFKA_INVENTORY_INSTANCE_EVENT_TOPIC:inventory.instance_ingress}
```

Comment:
I think this should be KAFKA_INVENTORY_INSTANCE_INGRESS_EVENT_TOPIC to match with line 62 of readme file.

Reply:
Yep, it's a mistake. Fixed, thanks!